src/components/Search/Search.js

```diff
@@ -72,7 +80,9 @@ function Search({
                   searchService.select({ ...items[0], section });
                 }
               } else if (key === 'ArrowDown' || key === 'ArrowUp') {
-                searchService.highlightSection(); // for improved accessibility
+                if (typeof searchService.highlightSection === 'function') {
```

Comment:
Can be merged with the previous `else if`.

Reply:
Indeed!